packages/components/psammead-navigation/src/index.stories.jsx

```diff
@@ -146,6 +147,16 @@ const navigationStory = (
     ? AmpScrollableNavigation
     : CanonicalScrollableNavigation;
 
+  const [isScrollable, setIsScrollable] = useState(false);
+
+  useEffect(() => {
+    const mediaQueryList = window.matchMedia('(max-width: 37.5rem)');
+    setIsScrollable(mediaQueryList.matches);
+    mediaQueryList.addListener(event => {
+      setIsScrollable(event.matches);
+    });
```

Comment:
With this refactor we don't have the `removeListener` method anymore.

```
    const handler = event => setMatches(event.matches);

    mediaQueryList.addListener(handler);
    return () => mediaQueryList.removeListener(handler);
```

Reply:
Good point, I'll add this back in for the story to prevent the storybook slowing down due to there being a load of extra listeners that aren't being used anymore.